tests/data/collapse-perf/inline-counter.txt

```diff
@@ -0,0 +1,243 @@
+inline-counter 26037  8801.193547:     250000 cpu-clock:uhH: 
```

Comment:
I'm not entirely sure why this file seems to only have been added now?

Reply:
I moved the file and had to update the path to the binary everywhere it was referenced.

Reply:
Ah, I just missed its removal is all!